knimin/lib/data_access.py

```diff
@@ -504,6 +507,8 @@ def format_survey_data(self, md):  # noqa
         -------
         dict of dict of dict
```

Comment:
Agree, this will probably be 5-10 ancillary functions in the end, so will be good to know what each dict holds.